WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -24,3 +24,20 @@ final class ShippingLabelFormViewModel {
         return [Section(rows: rows)]
     }
 }
+
+// MARK: - Utils
+extension ShippingLabelFormViewModel {
+    func fromAddressToShippingLabelAddress(address: Address?) -> ShippingLabelAddress? {
+        guard let address = address else { return nil }
+        let shippingLabelAddress = ShippingLabelAddress(company: address.company ?? "",
+                                                        name: address.firstName + " " + address.lastName,
```

Comment:
this naming rule wouldn't work for certain languages, as the order is often reversed in a few Asian languages. if we want to support localized name correctly, we probably want to follow:

https://github.com/woocommerce/woocommerce-ios/blob/6fef0340b60f8819cb86a7663afd5eee0997d7ba/WooCommerce/Classes/Model/Address+Woo.swift#L13-L17

Reply:
Wow, definitely TIL something new. 💯 
I made all the changes related to this suggestion here 51a3d8630f47b9f8d5978eaa1a609047b786d189